src/allianceutils/templatetags/common.py

```diff
@@ -0,0 +1,177 @@
+from collections.abc import Mapping
```

Comment:
i think this should move to `util/template.py`, and leave `tempaltetags/*` for actual tags